Data/Source/Scripts/Survival_MCM.psc

```diff
@@ -78,6 +78,14 @@ Event OnInit()
 EndEvent
 
 Event OnPageReset(String a_page)
+	
+	If(a_page == "")
+        LoadCustomContent("SurvivalLogo.dds")
+        return
+    Else
+        UnloadCustomContent()
+	EndIf
+	
```

Comment:
Let's make sure we're using tabs everywhere for consistency. Also I tend to put the control keywords in lowercase, and don't use parentheses when not needed.

Reply:
Yeah thats odd on my end the tabbing is set the way its supposed to be. Im not sure why that would have changed when I pushed the repo. Sorry for the late reply its been a busy week. 

As for the texture, thats a resolution issue. I have a fixed version I just pushed the wrong copy of the texture somehow up to the repo.

Ill put up another commit soon that has the correct version.